1_3.js

```diff
@@ -0,0 +1,21 @@
+let arr = ['掃除', '買い物', '散歩'];
+
+let print = (arr) => {
+  console.log("========================");
+  console.log("現在持っているタスク一覧");
+  console.log("========================");
+  for (let i = 0; i < arr.length; i++) {
+    console.log(i + ':' + arr[i]);
```

Comment:
コードとしては問題ないのですが
よりイマドキかつ、メンテナンスしやすい書き方をするならテンプレート文字列を使う方がよりよいかと思います。

```suggestion
    console.log(`${i} : ${arr[i]}`);    
```